fs/xfstests.py.data/xfstests.yaml

```diff
@@ -9,8 +8,15 @@ setup:
     # Run with either loop_type (or) disk_type
     loop_type: !mux
         type: 'loop'
-        loop_size: '5GiB'
-        disk: /dev/sdj2
+        loop_size: '7GiB'
+        # Option to provide disk for loop device creation,
+        # Uses '/' by default for file creation
+        # disk: /dev/sdj2
```

Comment:
`disk` can be added as a parameter here, without any value to it, so that it is easier to edit the value for automated runs later. Example below:
`disk: `